sql/wsrep_trans_observer.h

```diff
@@ -375,7 +398,7 @@ static inline int wsrep_after_commit(THD* thd, bool all)
               wsrep_is_active(thd),
               (long long)wsrep_thd_trx_seqno(thd),
               wsrep_has_changes(thd));
-  DBUG_ASSERT(wsrep_run_commit_hook(thd, all));
+  //DBUG_ASSERT(wsrep_run_commit_hook(thd, all));
```

Comment:
Let's not have assertions that are commented out. Either remove this line altogether, or put it back and fix if necessary.